bindings/ch/chsir.h

```diff
@@ -41,7 +41,7 @@ extern void* _Chsir_handle = dlopen("libsir.dl", RTLD_LAZY);
 int
 _chsir_early_abort(void) {
     (void)fprintf(_stderr, "Error: dlopen(): %s"
-                  "\n       Cannot get _Chsir_handle in %s:%d\n",
+                  "\x0d\x0a       Cannot get _Chsir_handle in %s:%d\x0d\x0a",
```

Comment:
This is in the C side of the binding, it doesn't include libsir until after this function, which is a Ch exec pragma abort handler, so you can't use SIR_EOL or any sir macros.  It also has to display correctly no matter the terminal mode, since the user might have set the mode to raw, so it should have the newline.